src/lib/viewers/controls/media/TimeControls.scss

```diff
@@ -0,0 +1,39 @@
+@import '../slider/styles';
+
+$bp-TimeControls-size: 18px; // Divisible by 3px (default) and 6px (hovered) for better vertical centering
```

Comment:
Is there some sort of grid styling that we can reuse here? For example, in box-ui-elements there is a variable called `$bdl-grid-unit` which is defaulted to 4px. Is the 3px above a constant that is being shared across multiple elements in the control bar?

Reply:
No, we don't have such a system for the preview controls. We're trying to keep the visual styles as close as possible to what's live today. We can investigate refactoring the styles if and when we do a visual refresh in the future.